data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -798,6 +798,68 @@
   ]
 },{
   "type" : "recipe",
+  "result": "mutagen_lamia",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_MUTAGEN",
+  "skill_used": "cooking",
+  "skills_required": [ "firstaid", 1 ],
+  "difficulty": 9,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "recipe_maiar", 8 ]],
+  "qualities":[
+    {"id":"CHEM","level":2,"amount":1}
+  ], "tools": [
```

Comment:
`"amount":1` is redundant.
The line above can be condensed into one line (ie. qualities and the one quality in it). Otherwise "tools" should be on a new line.